component/tests/common.py

```diff
@@ -105,7 +104,7 @@ def setUp(self):
 
 
 class ComponentRegistryCase(
-    unittest.TestCase, common.MetaCase("DummyCase", (object,), {})
+    common.TreeCase, common.MetaCase("DummyCase", (object,), {})
```

Comment:
With the other change mentioned, I think this should be left as is... The docstring in this class explains that this is being created for a specific purpose, so maybe it is best to leave it, even when Odoo's `TreeCase` inherits from `unittest.TestCase`...

A better solution would be not to use this class downstream where we don't need, but instead use one of Odoo's `common` cases.

Reply:
this class should not inherit from any odoo class. It's a base class for tests class like `TransactionComponentRegistryCase` or `SavepointComponentRegistryCase`.